### ISSUE TITLE
Template rendering fails with ytt 0.27.0

### ISSUE THREAD

**Issue by zankich** [cloudfoundry/cf-for-k8s#131]
If you render the templates with ytt 0.27.0 you get the following error:
```
ytt: Error:
- library.eval: Evaluating library 'github.com/cloudfoundry/uaa':
    in <toplevel>
      uaa.yml:195 | #@ uaa_resources = uaa.with_data_values(uaa_values()).eval()
    reason:
     - data.read: Expected to find file 'log4j2.properties' (hint: only files included via -f flag are available)
         in <toplevel>
           _ytt_lib/github.com/cloudfoundry/uaa/k8s/templates/config.yml:11 |   log4j2.properties: #@ data.read("log4j2.properties")
```

Template rendering succeeds with ytt 0.26.0.

**Comment by cppforlife** [comment]
shipped v0.27.1 (https://github.com/k14s/ytt/releases/tag/v0.27.1) to switch back semantics of data.read(). i wanted to make a breaking change but ultimately decided to support existing usage as well.

**Comment by zankich** [comment]
I can confirm that this is fixed with ytt 0.27.1, thanks!
